feature-swap-impl/src/main/java/io/novafoundation/nova/feature_swap_impl/domain/validation/SwapValidations.kt

```diff
@@ -38,6 +41,25 @@ fun SwapValidationSystemBuilder.swapSmallRemainingBalance(
     )
 )
 
+fun SwapValidationSystemBuilder.sufficientBalanceConsideringConsumersValidation(
+    assetSourceRegistry: AssetSourceRegistry
+) = sufficientBalanceConsideringConsumersValidation(
+    assetSourceRegistry,
+    chainExtractor = { it.detailedAssetIn.chain },
+    assetExtractor = { it.detailedAssetIn.asset.token.configuration },
+    totalBalanceExtractor = { it.detailedAssetIn.asset.totalInPlanks },
+    feeExtractor = { it.swapFee.networkFee.amount },
```

Comment:
Same comment about total fee as above

Reply:
Since I use the next check below in validation considering that asset is commission asset I think we not needed to use totalFee in feeExtractor. But it seems that `asset.isCommissionAsset` it's an unnecessary check and it's better to remove it and use your advice. What do you think?
```
if (asset.isCommissionAsset && totalCanDropBelowMinimumBalance) {
    ...
}
```

Reply:
Yeah `asset.isCommissionAsset` seems redundant, we can rely on `totalCanDropBelowMinimumBalance` entirely